app/views/admin/member_search/index.html.haml

```diff
@@ -0,0 +1,23 @@
+.container.py-4.py-lg-5
+  .row.mb-4
+    .col
+      %h1 Member search
+
+  .row
+    .col.col-md-10.col-lg-8
+      = render partial: 'search_form'
+
+  .row 
+    .col.col-md-10.col-lg-8
+      - if @members.present?
```

Comment:
We pass in named locals in the controller to this view. And, all views and partials.

Here, we should use that `members` local variable, not the `@members` instance variable.

The benefit is that each partial or view is told explicitly which variables it should have access to. And the errors the developer sees are NameErrors, not NoMethodError on nil (an @instance variable is nil if unset).

Reply:
So, we need to pass in the local variable name `members` to this view, from the controller action #index.

Long story short: either one uses instance variables all over, OR locals all over.